pkg/controller/pod_driver.go

```diff
@@ -968,13 +973,13 @@ func (p *PodDriver) needCreateMountPod(ctx context.Context, uniqueId, hashVal st
 		log.Error(err, "List pod error")
 		return false, err
 	}
-	needCreate := true
+	notAvailable := true
 	for _, po := range pods {
 		if po.DeletionTimestamp == nil && !resource.IsPodComplete(&po) {
-			needCreate = false
+			notAvailable = false
 		}
 	}
-	return needCreate, nil
+	return notAvailable, nil
```

Comment:
func name is has available mount, but return not available pod, It's very confusing.